common/src/test/kotlin/com.netguru.multiplatformstorage/MultiplatformStorageTest.kt

```diff
@@ -0,0 +1,232 @@
+package com.netguru.multiplatformstorage
+
+import kotlin.test.Test
+import kotlin.test.*
+
+expect val storage: MultiplatformStorage
+
+class MultiplatformStorageTest : BaseTest() {
+
+    //TODO 28.12.2018 Roboelectric doesn't support Keystore. Tests will be refactored in next task
```

Comment:
So you should then inject keystore instance with DI or hide it behind some additional abstraction.

Reply:
Code is tested with unit test, but we also want to have instrumentation/functional tests. This is not so easy to do that in multiplatform project so we have a separate task for that